src/stylesheets/components/_icon.scss

```diff
@@ -71,3 +71,7 @@ sds-icon.external-link > * > * {
     display: block;
   }
 }
+
+usa-icon.text-white {
+  color: #3f57a6 !important;
```

Comment:
Where is this hex code from? I'm not seeing it at one of our defined colors.

Reply:
@cwolf10 It's in the .postfix class search.component.scss

Reply:
Okay, not sure where that is coming from originally. Usually, the default is to stick with on one of the color tokens rather than go with a a hex code. Anyways, as I've already noted I think this shouldn't be updated as the solution, but just a note for future reference, try and make sure the class names that are being applied make sense. This class name indicates it is applying white to text, however #3f57a6 is blue. If that hex-code needs to be applied, a better solution would be to create a new class applying that hex code and then update the search component to apply that new class rather than .text-white

Reply:
@cwolf10  there was a Production issue at that time is was displaying black on sam.gov even though we have white icon. To fix the prod issue applied that class.

Reply:
@yerramshilpa  Okay, was the input made black or something? The icon should be black by default.

@julieduong22  Anyways, if we do have a use case that requires a white "X" icon then we should tailor the application of that class in search component even more than what we have now. It looks like presently if you just set `isSuffixSearchIcon` to true the text-white class is applied, even though that option does not make the input dark. Again, redefining usa-icon.text-white to be a blue color is not the way to fix this. It will cause the "X" to be visible again in this exact instance. But it sounds like it will reintroduce the issue Shilpa had resolved, and it will add additional defects by making all icons that have the text-white class applied to them to appear blue throughout the application.

Reply:
Hey @julieduong22, following up on this as I hadn't seen a PR adjusting search component. Did my last comment make sense as far as why adjusting a class definition, or based on your latest update, adding a new class won't fix the issue presented?